cms/djangoapps/contentstore/rest_api/v1/serializers/course_waffle_flags.py

```diff
@@ -40,9 +40,13 @@ def get_course_key(self):
 
     def get_use_new_home_page(self, obj):
         """
-        Method to get the use_new_home_page switch
+        Method to indicate whether we should use the new home page.
+
+        This used to be based on a waffle flag but the flag is being removed so we
+        default it to true for now until we can remove the need for it from the consumers
+        of this serializer and the related APIs.
         """
-        return toggles.use_new_home_page()
+        return True
```

Comment:
@feanil Just checking my understanding: this PR is a partial removal that ensures that the legacy page can no longer be accessed, and we'd plan to do a more thorough removal later? I'm OK with that approach, I'd just ask for some more words in the PR description as to the followup plan and a link in this docstring to the depr issue so that future devs know that the legacy-home-related code is dead and OK to remove.